iron-ajax.html

```diff
@@ -291,7 +291,14 @@
     },
 
     get requestHeaders() {
-      var headers = Object.create(this.headers || {});
+      var headers = {},
+          header;
+
+      if(this.headers instanceof Object) {
```

Comment:
That should work! I thought the `Object.create` call was for isolation purposes I didn't see, so I tried to stay close to this behavior. Will fix.